### ISSUE TITLE
Warning in CollectionParameters

### ISSUE THREAD

**Issue by a-brandt** [arangodb/elasticsearch-river-arangodb#2]
Hello,

there is a new attribute "count" in the collection result (ArangoDB 2.6).

The error was reported in the ArangoDB google group:
https://groups.google.com/forum/#!search/arangodb/arangodb/c2VjUf1O0is/DtlO3GPKbEgJ

**Comment by stackmagic** [comment]
Stacktrace:

```
[2015-05-13 20:08:56,571][WARN ][org.elasticsearch.river.arangodb.wal.states.CollectionCheck] Error getting inventory
com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "count" (class net.swisstech.arangodb.model.CollectionParameters), not marked as ignorable (10 known properties: "planId", "isVolatile", "name", "waitForSync", "maximalSize", "version", "doCompact", "deleted", "type", "cid"])
 at [Source: buffer(com.squareup.okhttp.internal.http.HttpConnection$FixedLengthSource@176abd4).inputStream(); line: 1, column: 79] (through reference chain: net.swisstech.arangodb.model.Inventory["collections"]->java.util.ArrayList[0]->net.swisstech.arangodb.model.ArangoDbCollection["parameters"]->net.swisstech.arangodb.model.CollectionParameters["count"])
        at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:51)
        at com.fasterxml.jackson.databind.DeserializationContext.reportUnknownProperty(DeserializationContext.java:817)
        at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:954)
        at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1315)
        at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1293)
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:249)
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:136)
        at com.fasterxml.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:523)
        at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:95)
        at com.fasterxml.jackson.databind.deser.impl.BeanPropertyMap.findDeserializeAndSet(BeanPropertyMap.java:285)
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:248)
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:136)
        at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:240)
        at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:212)
        at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:25)
        at com.fasterxml.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:523)
        at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:95)
        at com.fasterxml.jackson.databind.deser.impl.BeanPropertyMap.findDeserializeAndSet(BeanPropertyMap.java:285)
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:248)
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:136)
        at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:3560)
        at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:2646)
        at net.swisstech.arangodb.WalClient.inventory(WalClient.java:51)
        at net.swisstech.arangodb.WalClient.inventory(WalClient.java:42)
        at org.elasticsearch.river.arangodb.wal.states.CollectionCheck.execute(CollectionCheck.java:50)
        at org.elasticsearch.river.arangodb.wal.WalReaderRunnable.run(WalReaderRunnable.java:40)
        at java.lang.Thread.run(Thread.java:745)
```

**Comment by fceller** [comment]
collection parameters have an additional field "count" now.

**Comment by stackmagic** [comment]
Alright, the integration tests work and it's released as version 1.0.1 here: https://github.com/arangodb/elasticsearch-river-arangodb/releases/tag/v1.0.1

Please verify. This should fix the problem.
